doc/development-workflow.md

```diff
@@ -0,0 +1,22 @@
+# Development workflow
+
+## PR author
+
+1. Open a PR with the changes you want to merge.
+
+2. Add the "Waiting on factcheck" label if the changes require feedback from someone other than developers.
+
+3. Move the associated Trello card from "In progress" to "Code review".
```

Comment:
I _think_ @chrisroos already knows about the change in order of the workflow, but is unsure if/when we should add the "Waiting on factcheck label". Is that right, Chris?

Since the PR author is the person who knows whether or not the work needs fact-checking, it makes sense to me that the label is added in point 2 as Chris has suggested (i.e. when the PR is created). Otherwise cards might end up with the "LGTM" label and get merged before someone realises it needs fact-checking.

Maybe we should change the label to "Needs fact-checking"?

Having said all that, I don't really mind how we do this as long as we all have a shared understanding.